### ISSUE TITLE
Python 3.6 ttkthemes not working

### ISSUE THREAD

**Issue by sunnysid3up** [pyinstaller/pyinstaller#3517]
**System**
Windows 10
Python 3.6.4
PyInstaller 3.4.dev0+1033a8770

**Imports**
```
import time
import webbrowser
import ctypes
from ttkthemes import ThemedStyle
from tkinter import *
from tkinter.ttk import *
from configparser import ConfigParser
```

**Command** 
`pyinstaller main.spec`

**Spec File**
```
# -*- mode: python -*-
a = Analysis(['main.py'],
             pathex=['C:\\Users\\Computer\\AppData\\Local\\Programs\\Python\\Python36',
             'C:\\Users\\Computer\\AppData\\Local\\Programs\\Python\\Python36\\Scripts',
             'C:\\Users\\Computer\\AppData\\Local\\Programs\\Python\\Python36\\Lib\\ctypes',
             'C:\\Users\\Computer\\AppData\\Local\\Programs\\Python\\Python36\\Lib\\tkinter',
             'C:\\Users\\Computer\\AppData\\Local\\Programs\\Python\\Python36\\Lib\\site-packages\\ttkthemes',
             'C:\\Users\\Computer\\Documents\\MyApp'],
             hiddenimports=[],
             hookspath=None,
             runtime_hooks=None)
a.datas += [('pic1.ico','C:\\Users\\Computer\\Documents\\MyApp\\Images\\pic1.ico','DATA'),
            ('pic2.ico','C:\\Users\\Computer\\Documents\\MyApp\\Images\\pic2.ico','DATA'),
            ('pic3.ico','C:\\Users\\Computer\\Documents\\MyApp\\Images\\pic3.ico','DATA'),
            ('pic4.ico','C:\\Users\\Computer\\Documents\\MyApp\\Images\\pic4.ico','DATA'),
            ('pic5.ico','C:\\Users\\Computer\\Documents\\MyApp\\Images\\pic5.ico','DATA'),
            ('pic6.ico','C:\\Users\\Computer\\Documents\\MyApp\\Images\\pic6.ico','DATA')]
pyz = PYZ(a.pure)
exe = EXE(pyz,
          a.scripts,
          a.binaries,
          a.zipfiles,
          a.datas,
          name='App.exe',
          debug=False,
          strip=None,
          upx=False,
          console=False, icon='C:\\Users\\Computer\\Documents\\MyApp\\Images\\pic1.ico')
```

Hello, 
My executable fails to execute script. When I ran the same command after removing `from ttkthemes import ThemedStyle` from my script, my executable ran without any problems. I looked at the warning text file, and the only difference when I include ttkthemes, is the one listed below. I am using python 3, so I'm not sure why Tkinter module is needed. How do I fix this issue? Let me know if you need any more information from me. Thank you in advance! 

**Warning Text File**
`missing module named Tkinter - imported by ttkthemes.themed_tk (conditional), PIL.ImageTk (conditional)`

**Comment by snippsat** [comment]
You look at this [post](https://python-forum.io/Thread-Tkinter-How-to-make-pyinstaller-import-the-ttk-theme?pid=31159#pid31159) where did help someone with this.

**Comment by htgoebel** [comment]
Closes by  eebbb3b
